firebase-firestore/src/main/java/com/google/firebase/firestore/core/Bound.java

```diff
@@ -64,8 +65,8 @@ public String canonicalString() {
     } else {
       builder.append("a:");
     }
-    for (FieldValue indexComponent : position) {
-      builder.append(indexComponent.toString());
+    for (Value indexComponent : position) {
+      builder.append(ProtoValues.canonicalId(indexComponent));
```

Comment:
While we're fixing the format here, should we separate these values with something?

(Optional; may not be worth it. Just a thought because it may make these more readable.)

Reply:
Added comma separation.